5calls/app/src/main/res/values/strings.xml

```diff
@@ -365,4 +368,13 @@
     <!-- Action for a snackbar or button which brings you to the location update screen [CHAR_LIMIT=10] -->
     <string name="update">update</string>
 
+    <!-- URL to sign up for the 5 Calls newsletter [CHAR_LIMIT=NONE] -->
+    <string name="newsletter_url" translatable="false">https://my.sendinblue.com/users/subscribe/js_id/2p22o/id/1</string>
```

Comment:
@nickoneill Do you think we could use a 5calls.org URL here in case you change the newsletter subscription URL / service at some point? Something like 5calls.org/newsletter that redirects to here server-side?